tests/arch/arm/arm_irq_advanced_features/src/arm_shared_isrs.c

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2022 The Chromium OS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/ztest.h>
+#include <zephyr/arch/cpu.h>
+#include <zephyr/arch/arm/aarch32/cortex_m/cmsis.h>
```

Comment:
again. This must not be aarch32 specific.

Reply:
The other tests in this directory are also aarch32 specific. Should this test be implemented in another place?

Reply:
yes, this should be moved to `tests/kernel/interrupt/`